### PR TITLE
small go post fixes

### PR DESCRIPTION
**Pull request by jadekler**
- Missing bracket
- codepb.CANCELED -> status.Canceled

### CODE REVIEW

**Comment by dfawley** [comment]
While you're at it, can you also change:

```go
clientDeadline := time.Now().Add(time.Duration(*deadlineMs) * time.Millisecond)
ctx, cancel := context.WithDeadline(ctx, clientDeadline)
```

to

```go
ctx, cancel := context.WithTimeout(ctx, time.Duration(*deadlineMs) * time.Millisecond)
```

**Comment by jadekler** [comment]
Done

File: _posts/2018-02-26-grpc-and-deadlines.md [hunk 1]
```diff
@@ -78,7 +78,8 @@ if (context->IsCancelled()) {
 
 ```go
 if ctx.Err() == context.Canceled {
```
Comment (dfawley) [inline _posts/2018-02-26-grpc-and-deadlines.md]: More complete would be:

```go
switch ctx.Err() {
  case context.Canceled:
    return status.New(codes.Canceled, "client cancelled, abandoning.")
  case context.DeadlineExceeded:
    return status.New(codes.DeadlineExceeded, "deadline exceeded")
  case nil:
    break
  default:
    return status.New(codes.Unknown, "unknown context error")
}
```

Hmm, maybe we should add a `status.FromContextError(error) *status.Status` or `status.ConvertContextError(error) error` convenience function that does this, so the user's code is:

```go
if err := ctx.Err(); err != nil {
  return status.FromContextError(error).Err()
}
```

Sadly, until 1.6 support is gone, we'll need 2 conditionally compiled files to do this so we can handle both "context" and "x/net/context" for go1.7 and go1.8.. but it's probably worth it.

**Comment by jadekler** [comment]
Right on. I actually don't have contributor access to this repo - could you squash + merge? Or, should I ask @hsaliak for access?

**Pull request merged**